netmanager/src/config/urls/analytics.js

```diff
@@ -24,3 +24,5 @@ export const DOWNLOAD_DATA = `${BASE_ANALYTICS_URL}/analytics/data/download?type
 export const GET_MONITORING_SITES_URI = `${BASE_ANALYTICS_URL}/analytics/dashboard/monitoringsites?organisation_name=KCCA`;
 
 export const GET_DATA_MAP = `${BASE_ANALYTICS_URL}/analytics/dashboard/monitoringsites?organisation_name=KCCA&pm25_category=`;
+
+export const URBAN_BETTER_DOWNLOAD_DATA_URI = `${BASE_ANALYTICS_URL}/analytics/data/download?tenant=urbanbetter`;
```

Comment:
I am just curious about the assignment of these tenant values. Do we expect to get them from some ENV or global state for that matter? But this is just a curiosity comment, no need to take any action right now. Thanks @Codebmk